setup.py

```diff
@@ -5,13 +5,21 @@
 
 setuptools.setup(
     name="jf_agent",
-    version="0.0.20",
+    version="0.0.21",
     description="An agent for collecting data for jellyfish",
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/Jellyfish-AI/jf_agent",
     packages=setuptools.find_packages(),
-    install_requires=["jira", "tqdm", "stashy", "pyyaml>=4.2b1", "urllib3==1.24.2", "dateparser"],
+    install_requires=[
+        "jira",
+        "tqdm",
+        "stashy",
+        "pyyaml>=4.2b1",
+        "urllib3==1.24.2",
+        "dateparser",
+        "pygithub @ git+ssh://git@github.com/Jellyfish-AI/PyGithub@ed5ec0ab#egg=pygithub-1.43.2",
```

Comment:
I don't follow the syntax here -- what's ed5ec0ab? I don't see that hash in our fork of the repo. And what does it mean to have the #egg tag?

Reply:
https://python-packaging.readthedocs.io/en/latest/dependencies.html#packages-not-on-pypi

Reply:
That hash seems to be the commit on my branch, not the commit as merged into master.  I'll change this to the master one for clarity.